sections/desktop-img.liquid

```diff
@@ -0,0 +1,23 @@
+{% for block in section.blocks %}
+  <div class="desktop-images">
+    <img src="{{ block.settings.desktop_image_1 | img_url: '800x' }}">
+  </div>
+{% endfor %}
```

Comment:
@BarretoAaron with this code, you will add 3 times the `desktop-images` container?

something like this

```html
<div class="desktop-images">
  <img src="{{ block.settings.desktop_image_1 | img_url: '800x' }}">
</div>
<div class="desktop-images">
  <img src="{{ block.settings.desktop_image_1 | img_url: '800x' }}">
</div>
<div class="desktop-images">
  <img src="{{ block.settings.desktop_image_1 | img_url: '800x' }}">
</div>
```

I'm asking that because this changes your initial structure, isn't it?

```html
<div class="desktop-images">
  <img src="{{ block.settings.desktop_image_1 | img_url: '800x' }}">
  <img src="{{ block.settings.desktop_image_1 | img_url: '800x' }}">
  <img src="{{ block.settings.desktop_image_1 | img_url: '800x' }}">
</div>
```

Reply:
Then it should be something like this?
`  <div class="desktop-images">  
</div>
{% for block in section.blocks %}
  <img src="{{ block.settings.desktop_image_1 | img_url: '800x' }}">
{% endfor %}`